src/menu/menu.scss

```diff
@@ -19,89 +19,141 @@
+    + .iui-icon {
+      margin-right: 0;
+      margin-left: $iui-s;
+    }
+  }
+
+  .iui-menu-description {
+    display: none;
+    font-size: $iui-font-size-small;
     @include themed {
-      color: t(iui-text-color);
+      opacity: t(iui-opacity-3);
     }
+  }
 
-    > .iui-icon {
-      @include iui-menu-icon;
+  > .iui-icon {
+    @include iui-menu-icon;
+  }
+
+  &:hover:not(.iui-disabled):not(.iui-active) {
+    @include themed {
+      background-color: rgba(
+        t(iui-color-foreground-primary-rgb),
+        t(iui-opacity-6)
+      );
     }
+  }
 
-    > .iui-content {
-      flex: 1 1 auto;
-      white-space: nowrap;
-      overflow: hidden;
-      text-overflow: ellipsis;
-      height: $iui-line-height;
+  &:focus {
+    @include themed {
+      outline: 1px solid t(iui-color-foreground-primary);
+      outline-offset: -1px;
+    }
+  }
 
-      + .iui-icon {
-        margin-right: 0;
-        margin-left: $iui-s;
-      }
+  &:focus:not(:focus-visible) {
+    outline: none;
+  }
+
+  &.iui-large {
+    height: $iui-component-height-large;
+
+    .iui-menu-description {
+      display: block;
+    }
+  }
+
+  &.iui-active {
+    @include themed {
+      background-color: rgba(
+        t(iui-color-foreground-primary-rgb),
+        t(iui-opacity-5)
+      );
+      border: solid 1px t(iui-color-foreground-primary);
     }
 
-    &:hover:not(.iui-disabled):not(.iui-active),
-    &:focus:not(.iui-disabled):not(.iui-active) {
+    > .iui-icon {
       @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        );
+        fill: t(iui-icons-color-primary);
       }
     }
 
     &:focus {
-      outline: none;
+      outline-offset: -$iui-xxs;
     }
+  }
 
-    &.iui-active {
-      box-sizing: border-box;
-      padding-top: $iui-component-padding-vertical;
-      padding-bottom: $iui-component-padding-vertical;
-      @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-5)
-        );
-        border: solid 1px t(iui-color-foreground-primary);
-      }
-
-      > .iui-icon {
-        @include themed {
-          fill: t(iui-icons-color-primary);
-        }
-      }
+  &.iui-disabled {
+    cursor: not-allowed;
+    @include themed {
+      color: t(iui-text-color-placeholder);
     }
 
-    &.iui-disabled {
-      cursor: not-allowed;
+    > .iui-icon {
       @include themed {
-        background-color: t(iui-color-background-disabled);
-        color: t(iui-text-color-muted);
-      }
-  
-      > .iui-icon {
-        @include themed {
-          fill: t(iui-icons-color-actionable-disabled);
-        }
+        fill: t(iui-icons-color-actionable-disabled);
       }
     }
   }
 }
 
+@mixin iui-menu-divider {
+  height: 1px;
+  width: calc(100% - #{$iui-l});
+  margin: auto;
+  @include themed {
+    background-color: t(iui-color-background-4);
+  }
+}
+
 @mixin iui-menu-icon {
```

Comment:
Spacing is missing.
![image](https://user-images.githubusercontent.com/36186912/124082282-31f2c080-da55-11eb-9854-0990f2ecb98d.png)

Reply:
Yikes, that's embarrassing.  It is fixed with the next push.